packages/xsel/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE=http://www.kfish.org/software/xsel/
+TERMUX_PKG_DESCRIPTION="Command-line program for getting and setting the contents of the X selection"
+TERMUX_PKG_LICENSE="custom"
```

Comment:
Xsel was released under MIT license (as majority of X packages) so `TERMUX_PKG_LICENSE="MIT"`.